addons/account/views/account_reconcile_model_views.xml

```diff
@@ -24,11 +24,11 @@
                             <field name="show_force_tax_included" invisible="1"/>
                             <field name="force_tax_included"
                                    attrs="{'invisible': [('show_force_tax_included', '=', False)]}" force_save="1"/>
-                            <!-- TODO widget="analytic_distribution" -->
                             <field name="analytic_distribution_stored_char" invisible="1"/>
-                            <field name="analytic_distribution" invisible="1" groups="analytic.group_analytic_accounting"
-                                                       options="{'product_field': 'product_id', 'account_field': 'account_id'}"
-                                                       business_domain_compute="parent.move_type in ['out_invoice', 'out_refund', 'out_receipt'] and 'sale' or parent.move_type in ['in_invoice', 'in_refund', 'in_receipt'] and 'purchase' or 'general'"/>
+                            <field name="analytic_distribution" widget="analytic_distribution"
+                                   groups="analytic.group_analytic_accounting"
+                                   options="{'account_field': 'account_id'}"
+                                   business_domain_compute="parent.move_type in ['out_invoice', 'out_refund', 'out_receipt'] and 'invoice' or parent.move_type in ['in_invoice', 'in_refund', 'in_receipt'] and 'bill' or 'general'"/>
```

Comment:
What is the move type on reconcile models?
And what is the parent on the line's form view?

Reply:
idem

Reply:
How is this one even working? :sweat_smile:

Reply:
It wasn't before, it was in invisible. I copy-pasted too fast for this field. Fixed it